speakerbox/src/test/java/com/mapzen/speakerbox/SpeakerboxTest.java

```diff
@@ -124,6 +124,14 @@ public void shouldReturnMuteState() throws Exception {
     }
 
     @Test
+    public void shouldStopOnMute() {
```

Comment:
Good catch. I like your second thought...it looks like all TextToSpeech#stop() does is stop the service so definitely no harm in always calling it.

http://androidxref.com/6.0.1_r10/xref/frameworks/base/core/java/android/speech/tts/TextToSpeech.java